lib/constants.js

```diff
@@ -36,7 +36,9 @@ var Constants = {
         AAD_API_VERSION : 'api-version',
         USERNAME : 'username',
         PASSWORD : 'password',
-        REFRESH_TOKEN : 'refresh_token'
+        REFRESH_TOKEN : 'refresh_token', 
+        LANGUAGE : 'mkt', 
+        DEVICE_CODE : 'device_code', 
```

Comment:
it's not a grant type, this is one of the fields returned when calling to get usercodeinfo.